config/connect-avro-distributed.properties

```diff
@@ -57,7 +57,26 @@ internal.value.converter=org.apache.kafka.connect.json.JsonConverter
 internal.key.converter.schemas.enable=false
 internal.value.converter.schemas.enable=false
 
-# Confuent Control Center Integration -- uncomment these lines to enable Kafka client interceptors
+# Confluent Control Center Integration -- uncomment these lines to enable Kafka client interceptors
 # that will report audit data that can be displayed and analyzed in Confluent Control Center
 # producer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
 # consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+
+# These are provided to inform the user about the presence of the REST host and port configs
```

Comment:
I was aiming to keep divergence from kafka/config/connect-[standalone,distributed].properties to a minimum. Thus, I ported the exact lines as they were at the end of the file. This is not ideal. Normally the two files should be exactly the same minus the differences. What would be your suggestion?

a) Diverge, b) reflect as is for now without improvements, c) change here as well as in AK on a separate PR that will be merged later.